### PR TITLE
Added copied content for review

### PR DESCRIPTION
**Pull request by ernestkorir**
Copied content from my previous project to do a review of it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <header>Js Documentation</header>
+      <a class="nav-link" href="#Introduction" >Introduction</a>
+      <a class="nav-link" href="#What_you_should_already_know" >What you should already know</a>
+      <a class="nav-link" href="#JavaScript_and_Java" >JavaScript and Java</a>
+      <a class="nav-link" href="#Hello_world" >Hello world</a>
+      <a class="nav-link" href="#Variables" >Variables</a>
+
```
Comment (ernestkorir) [inline index.html:18]: You should remove this empty line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <p>This is nineth paragraph</p>
+        <p>This is tenth paragraph</p>
+        <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM).</li>
+        <code>var x = 42.</code>
+      </article>
+    </section>
+  
+  </main>
+
+
```
Comment (ernestkorir) [inline index.html:71]: Remove unnecessary empty lines

File: index.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      <article>
+        <p>This is first paragraph</p>
+        <p>This is second paragraph</p>
+        <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM).</li>
+        <code>const MY_OBJECT = {"key": "value"}; MY_OBJECT.key =
+          "otherValue";</code>
+      </article>
+    </section>
+    <section class="main-section" id="What_you_should_already_know">
+      <header>What you should already know</header>
```
Comment (ernestkorir) [inline index.html:32]: You can use hx that is h1-h6 here

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ernestkorir    , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to phrase the comments politely, as suggestions instead of orders.
For example:
    ### _"I think it would be better to use semantic tags"_

- [ ] Kindly make sure to always includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: 
    ### _*Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ernestkorir) [reply index.html:18]: I think  it's much better to remove this empty line for the code to look neat

File: index.html (hunk 2 above)
Comment (ernestkorir) [reply index.html:71]: I think here too it looks better without these empty lines

File: index.html (hunk 3 above)
Comment (ernestkorir) [reply index.html:32]: I suggest instead of using a header tag kindly use a hx tag that is h1-h6 depending on the size you want

**Review by emmiiorji — APPROVED** [review]
Hi @ernestkorir, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Good Highlights✔

- Good use of inline comments
- Review comments are polite
- Requested changes have been implemented

## Optional suggestions

- Using a couple of emojis and/or gifs in the review can make the code review more human-friendly 👽🤖.
   I recommend you use it.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**